en/simulation/multi-vehicle-simulation.md

```diff
@@ -62,7 +60,7 @@ For each simulated vehicle, the following is required:
 The launch file `multi_uav_mavros_sitl.launch`does the following,
 
 * loads a world in gazebo,
-```
+  ```
```

Comment:
@lamping7
Note
1. Typically use three backticks on a new line for a block of code and single backticks for an inline code. With a block you can specify the programming language for syntax highlighting (I have not done this here).
2. The backticks and blocks have to be indented next to the text in the bullet - in this case 2 spaces.